### ISSUE TITLE
Why did price change without notice? 

### ISSUE THREAD

**Issue by becz03** [liveask/liveask#22]
I'm feeling a bit extorted right now... I used this website various times in the last 6 months at $0 and the event expired after much longer than 7 days, and there was no fee - I could still see the submitted questions after expiration, even if the link wasn't active. Now I suddenly go to check on my team's submitted questions for a meeting next week where the President of our company is visiting the department, and all the questions are blurred out, and I can't ask an entire team to remember what they submitted weeks ago. And I can't see what's there without paying almost $7 per live-ask. 

I recreated an event today to see if I missed a warning sign and sure enough, in tiny letters on the event creation page, with zero arrows or bright colors or anything showing me that paragraph that I very easily could have missed - and did miss - it says it'll expire in 7 days. Now I have to pay you to see the content that used to last 30 days? How do I see these questions?

**Comment by extrawurst** [comment]
TLDR; Please reach out to me via `mail at live-ask.com` with the event link you refer to - if it was created with the 30 days timeout initially we can surely unlock it again
